modules/core/shared/src/main/scala/freestyle/tagless/internal/tagless.scala

```diff
@@ -46,13 +46,11 @@ object taglessImpl {
 
 }
 
-case class Algebra( clait: Clait) {
+case class Algebra( clait: Clait, isStackSafe: Boolean) {
   import clait._
   val errors = new ErrorMessages("@tagless")
   import errors._
 
-  val isStackSafe = clait.mods.isStackSafe
```

Comment:
The implementation in `ScalametaUtils.ModsOps` still exists; we should probably remove it unless we want to preserve `@stacksafe` support (in that case we need to check for either case in `isStackSafe`).